staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -170,6 +170,20 @@ type DomainSpec struct {
 	// One of: shared, auto
 	// +optional
 	IOThreadsPolicy *IOThreadsPolicy `json:"ioThreadsPolicy,omitempty"`
+	// Chassis specifies the chassins info passed to the domain.
+	// +optional
+	Chassis *Chassis `json:"chassis,omitempty"`
+}
+
+// Chassis specifies the chassins info passed to the domain.
+// ---
+// +k8s:openapi-gen=true
+type Chassis struct {
+	Manufacturer string `json:"manufacturer,omitempty"`
```

Comment:
Are each of these values mandatory? The way this is written, and the way the logic in converter.go is written, each of these fields will be incldued in the dom XML, even if they're empty.

Perhaps these values could be `*string` instead of `string`, then we can test for nil before including each element in converter.go. @vladikr can I get a second opinion on this? Am I making things too hard?

Reply:
If there are no values being passed, this is the output I'm seeing
```xml
  <sysinfo type="smbios">
    <system>
      <entry name="uuid">e4686d2c-6e8d-4335-b8fd-81bee22f4814</entry>
      <entry name="serial">e4686d2c-6e8d-4335-b8fd-81bee22f4815</entry>
    </system>
    <bios></bios>
    <baseBoard></baseBoard>
    <chassis></chassis>
  </sysinfo>
```

Reply:
I'm not sure. This seems fine to me.